tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.AssemblyReferencesHaveNotChanged.netcoreapp3.1.verified.txt

```diff
@@ -47,6 +47,7 @@ System.Runtime.Loader, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f
 System.Runtime.Numerics, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
 System.Runtime.Serialization.Formatters, Version=4.0.4.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
 System.Runtime.Serialization.Primitives, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
+System.Security.Claims, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
```

Comment:
We have to be careful here: We need to make sure that we only depend on the version that ships with the framework 😬 I'm surprised there's no changes for netstandard tho 🤔

Reply:
🤔 the version of framework is in mscorlib, maybe should ducktype here?
![image](https://github.com/DataDog/dd-trace-dotnet/assets/8353486/6073d593-e0a8-4125-bbf0-ff5640e4743b)

Reply:
maybe because `System.Security.Claims` is part of [netstandard already](https://learn.microsoft.com/en-us/dotnet/api/system.security.claims.claimsidentity?view=netstandard-2.1)?

Reply:
https://apisof.net/catalog/e2fe20ea-1312-3d06-d9b6-22e322fc1ceb

Reply:
thanks, so apart from mac catalyst, it all seems to be shipped with the framework 🤔

Reply:
What we have to be careful about is the _Version_ I think. As long as 4.1.2.0 is the version that ships with .NET Core 3.1, we're fine 🙂

Reply:
Confirmed: 

![image](https://github.com/DataDog/dd-trace-dotnet/assets/18755388/24733d94-db0b-482f-922a-5ddaba9bd48a)

you're in the clear 😉

Reply:
thank you!